mesheryctl/internal/cli/root/pattern/onboard_test.go

```diff
@@ -69,16 +69,16 @@ func TestOnboardCmd(t *testing.T) {
 				{
 					Method:       "POST",
 					URL:          testContext.BaseURL + "/api/pattern/deploy",
-					Response:     "onboard.applicationdeploy.response.golden",
+					Response:     "onboard.patterndeploy.response.golden",
 					ResponseCode: 200,
 				},
 			},
 			Token:       filepath.Join(fixturesDir, "token.golden"),
 			ExpectError: false,
 		},
 		{
-			Name:             "Onboard Application with --skip-save",
-			Args:             []string{"onboard", "-f", filepath.Join(fixturesDir, "sampleApp.golden"), "--skip-save", "-s", "Kubernetes Manifest"},
+			Name:             "Onboard patternlication with --skip-save",
```

Comment:
Please update them as below... 
```suggestion
			Name:             "Onboard pattern with --skip-save",
```

(IDK if you used CTRL+F2 in VSCode and updated it...)

Reply:
Typo fix. ✅